README.md

```diff
@@ -1 +1,14 @@
-Orbit
+# Exposure System
+
+### Description
+
+Exposure system is a new initiative to expose **Humans of AltCampus** to various different knowledge other than just programming, by providing them with articles and tasks on a regular basis.
```

Comment:
@Yjspoder @AmanSharma4419  "various different knowledge" doesn't feel right. Replace it with "various ideas and opinions"

Reply:
@prank7 Resolved